### PR TITLE
Completed html-forms. -- Joshua Preciado

### PR DESCRIPTION
**Pull request by joshuapreciado**
https://user-images.githubusercontent.com/78586776/151491216-5513497f-ed8b-4039-b9d0-edb9ddcf0233.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <div>
+      <label><input type="radio" name="favoritelanguage" value="html" checked="checked" /> HTML </label>
+      <label><input type="radio" name="favoritelanguage" value="CSS" /> CSS </label>
+      <label><input type="radio" name="favoritelanguage" value="javascript" /> JavaScript </label>
+    </div>
+    <div>
+      <p><label for="moreinfo">Would you like to sign up for more information?</label></p>
+    </div>
+    <input type="checkbox" name="moreinfo" value="Sign me up!" checked="checked" /> Sign me up!
+    <div></div>
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to use these empty divs to `wrap` the label and input elements, you don't want empty elements floating around.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  <h1>Coding Form!</h1>
+  <form action="index.html">
+    <label for="email">Email</label>
+    <div>
+      <input type="text" id="email" name="email" size="15" maxlength="30">
+    </div>
+    <div>
+      <label for="Why is Coding Cool?">Why is Coding Cool?</label>
+      <div>
+        <select name="reasons">
```
Comment (BrettAlbright) [inline html-forms/index.html]: need an id for this select to match the `for`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <label><input type="radio" name="favoritelanguage" value="javascript" /> JavaScript </label>
+    </div>
+    <div>
+      <p><label for="moreinfo">Would you like to sign up for more information?</label></p>
+    </div>
+    <input type="checkbox" name="moreinfo" value="Sign me up!" checked="checked" /> Sign me up!
+    <div>
+      <label for="Comments">Comments</label>
+    </div>
+    <textarea name="comments" cols="33" rows="12"></textarea>
```
Comment (BrettAlbright) [inline html-forms/index.html]: just need the `id` here for the text area to match the `for` attribute on the label

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
final update, great work!

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+    <div>
+      <label><input type="radio" name="favoritelanguage" value="html" checked="checked" /> HTML </label>
+      <label><input type="radio" name="favoritelanguage" value="CSS" /> CSS </label>
+      <label><input type="radio" name="favoritelanguage" value="javascript" /> JavaScript </label>
+    </div>
+    <div>
+      <p><label for="moreinfo">Would you like to sign up for more information?</label></p>
+    </div>
+    <input type="checkbox" name="moreinfo" value="Sign me up!" checked="checked" /> Sign me up!
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to add the matching `id` for the `for` attribute on the input

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <div>
+      <label for="Comments">Comments</label>
+    </div>
+    <textarea name="Comments" cols="33" rows="12"></textarea>
```
Comment (BrettAlbright) [inline html-forms/index.html]: same as above

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Just two tiny updates! nice work!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**